L1Trigger/TrackFindingTracklet/src/PurgeDuplicate.cc

```diff
@@ -184,19 +184,11 @@ void PurgeDuplicate::execute(std::vector<Track>& outputtracks, unsigned int iSec
           }
         }
 
-        // Initialize all-false 2D array of tracks being duplicates to other tracks
-        bool dupMap[numStublists][numStublists];  // Ends up symmetric
-        for (unsigned int itrk = 0; itrk < numStublists; itrk++) {
-          for (unsigned int jtrk = 0; jtrk < numStublists; jtrk++) {
-            dupMap[itrk][jtrk] = false;
-          }
-        }
+        // Initialize all-false 2D vector of tracks being duplicates to other tracks
+        vector<vector<bool>> dupMap(numStublists, vector<bool>(numStublists, false));
```

Comment:
If symmetric and storage is an issue why not using triangular storage?
btw interesting enough ```vector<bool>``` uses one bit per entry while bool v[N] will use a full byte...